deploy/example/storageclass-smb.yaml

```diff
@@ -13,6 +13,7 @@ parameters:
   csi.storage.k8s.io/node-stage-secret-name: "smbcreds"
   csi.storage.k8s.io/node-stage-secret-namespace: "default"
   createSubDir: "false"  # optional: create a sub dir for new volume
+  nobrl: "false"  # optional: add "nobrl" mount option; use with caution
```

Comment:
this option `nobrl` could be added under `mountOptions`, it's not necessary to add a new parameter for this option.

Reply:
so this driver already supports `nobrl`

Reply:
Ok, thanks.
Definitely something that I've overlooked.

Actually I'm pretty new to the topic of CSI and didn't know `mountOptions `would be the correct way to go.
I will give it a try and will close the PR if necessary.